Ip/Internal/Repository/assets/ipRepositoryAll.js

```diff
@@ -304,7 +377,6 @@
             }
 
             // filename
-            $file.find('span').text(data.fileName);
```

Comment:
Why this is removed?

Reply:
I had some custom code here, removed too much by accident